h/dynalloc.h

```diff
@@ -696,11 +741,23 @@ typedef struct DynallocInputParms_tag {
   char reserved[3];
 } DynallocInputParms;
 
+
+typedef struct DynallocNewInputParms_tag {
+  int type;
+  int size;
+  int key;
+  union {
+    int numValue;
+    char *stringValue;
+  } data;
+} DynallocNewInputParms;
+
 #pragma map(dynallocDataset, "DYNAUALC")
 #pragma map(dynallocDatasetMember, "DYNAUALM")
 #pragma map(unallocDataset, "DYNADALC")
 
 int dynallocDataset(DynallocInputParms *inputParms, int *reasonCode);
+int dynallocNewDataset(DynallocNewInputParms *setInputParms, int TextUnitsSize, int *reasonCode);
```

Comment:
On the one hand you've tried abstracting away text units by introducing `DynallocNewInputParms`, but on the other hand there is `TextUnitsSize`. 
* What does one need to pass in that argument?
* The name shouldn't start with a capital letter

In my opinion, we should either pass raw text units or pass the concrete data set attributes, not something in between.

Reply:
I have renamed variable name to `inputParmsCount` as it is `DynallocNewInputParms` count passed from caller. 
In `dynallocDataset` there are fixed 3 number of `inputParms` but in `dynallocNewDataset`  `inputParms` count is not fixed so this argument is required.